add hydra team update for week #24 (#187)
input-output-hk/cardano-updates: blog/2023-06-16-hydra.md

---
title: Hydra Team Update
slug: 2023-06-16-hydra
authors: ffakenz
tags: [hydra]
hide_table_of_contents: false
---

## High-level summary

This week, the Hydra team focused their efforts on continuing to investigate

> and troubleshoot the broken head on the mainnet. They collected several bugs

Reviewer: ```suggestion
and experiment operating a head on mainnet. They collected several bugs
```

and issues and worked on fixing them. Now the team is soon to release a new
version, 0.11.0, which comes with a lot of improvements and bug fixes.

## What did the team achieve this week

-   Fixed a bug on broken head [#927](https://github.com/input-output-hk/hydra/issues/927)
-   Restored a broken head on mainnnet

> -   Closed user issue [#914](https://github.com/input-output-hk/hydra/issues/914)

Reviewer: ```suggestion
-   Solved one user issue [#914](https://github.com/input-output-hk/hydra/issues/914)
```

> -   Don’t store full scripts in the local state [#928](https://github.com/input-output-hk/hydra/pull/928)

Reviewer: ```suggestion
-   Reduced significantly reduced local state size and logs by removing the full scripts from it [#928](https://github.com/input-output-hk/hydra/pull/928)
```

> -   Don’t send full transaction in the snapshot body, instead use tx id [#922](https://github.com/input-output-hk/hydra/pull/922)

Reviewer: ```suggestion
-   Reduced significantly reduced snapshot size in the API, removing the full transactions from it and only including their tx id [#922](https://github.com/input-output-hk/hydra/pull/922)
```

> -   Prevent seeing close from another head [#927](https://github.com/input-output-hk/hydra/issues/927)

Reviewer: Duplication of the first item

```suggestion
```
